### PR TITLE
scripts

### PR DESCRIPTION
**Pull request by tom-wa**
### info

```sh
% kdb mountpoint-info
Version: 0.8.19
Default resolver: resolver_fm_hpu_b
Default storage: ini

 % kdb mount /tmp/test.ini system/test ini hello=ini -c hello=world

 % kdb mountpoint-info system/test
Version: 0.8.19
Default resolver: resolver_fm_hpu_b
Default storage: ini
Mountpoint: system/test
File: /tmp/test.ini
        config:
        hello = world
        path = /tmp/test.ini
getplugins:
        #0#resolver
        #5#ini#ini#
                config:
                hello = ini
setplugins:
        #0#resolver
        #5#ini
        #6#sync#sync#
        #7#resolver
errorplugins:
        #5#resolver_fm_hpu_b#resolver#
```

### mount-list-all-files

```sh
 % kdb mount
none on system/info/elektra/constants with name system/info/elektra/constants
/usr/local/share/doc/elektra/CONTRACT.ini on system/info/elektra/contract/#0 with name system/info/elektra/contract/#0
none on system/info/elektra/desktop with name system/info/elektra/desktop
/usr/local/share/doc/elektra/METADATA.ini on system/info/elektra/metadata/#0 with name system/info/elektra/metadata/#0
none on system/info/elektra/uname with name system/info/elektra/uname
/tmp/test.ini on system/test with name system/test

 % kdb mount-list-all-files
none
/tmp/test.ini
/usr/local/share/doc/elektra/CONTRACT.ini
/usr/local/share/doc/elektra/METADATA.ini
```

### change-resolver-symlink
```
kdb change-resolver-symlink resolver_fm_b_b
```
### change-storage-symlink
```
kdb change-resolver-symlink ini
```
### check-env-dep
```
kdb check-env-dep
```
### restore
```
% kdb mount
% kdb mount-info
% kdb set user/x foo

Create a new key user/x with string foo
% kdb mount a.ini /a
% kdb mount
a.ini on /a with name /a
none on system/info/elektra/constants with name system/info/elektra/constants
/usr/local/share/doc/elektra/CONTRACT.ini on system/info/elektra/contract/#0 with name system/info/elektra/contract/#0
none on system/info/elektra/desktop with name system/info/elektra/desktop
/usr/local/share/doc/elektra/METADATA.ini on system/info/elektra/metadata/#0 with name system/info/elektra/metadata/#0
none on system/info/elektra/uname with name system/info/elektra/uname

% kdb backup
restore with "kdb restore 1494911664"

% kdb get user/x
Did not find key
% kdb mount

% kdb restore 1494911664
                                                                                                                                     [1]
% kdb get user/x
foo
% kdb mount
a.ini on /a with name /a
none on system/info/elektra/constants with name system/info/elektra/constants
/usr/local/share/doc/elektra/CONTRACT.ini on system/info/elektra/contract/#0 with name system/info/elektra/contract/#0
none on system/info/elektra/desktop with name system/info/elektra/desktop
/usr/local/share/doc/elektra/METADATA.ini on system/info/elektra/metadata/#0 with name system/info/elektra/metadata/#0
none on system/info/elektra/uname with name system/info/elektra/uname
```

### CODE REVIEW

File: doc/help/kdb-info_2.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+kdb-info_2 -- Print information about the default storage and resolver or a mountpoint
```
Comment (markus2330) [inline doc/help/kdb-info_2.md]: Great tool, but the name can be improved. What about mountpoint-info?

**Comment by tom-wa** [comment]
currently `backup` unmounts all backends but only backs up the default files for user, spec and system. is that sufficient or shall we backup all mounted files ?

File: doc/help/kdb-info_2.md (hunk 1 above)
Comment (tom-wa) [inline doc/help/kdb-info_2.md]: info_2 was only a placeholder, renamed to mountpoint-info

**Comment by markus2330** [comment]
If they are not mounted, we are not responsible for the files anymore, so it should be okay.

Can you document that please?

**Comment by markus2330** [comment]
Removed "ready for merge" and wait for a bit more docu.

**Comment by markus2330** [comment]
So great to finally have these scripts!

**Pull request merged**